.gitignore

```diff
@@ -37,3 +37,6 @@ dist
 ### Hardhat
 cache
 artifacts
+
+# Coverage reports
+**/coverage/*
```

Comment:
```suggestion
**/coverage/*

# We're using pnpm, ignoring `package-lock.json`
package-lock.json
```

Add it to `.gitignore` so that you don't accidentally install it again.